plugin/pkg/scheduler/metrics/metrics.go

```diff
@@ -68,3 +79,12 @@ func Register() {
 func SinceInMicroseconds(start time.Time) float64 {
 	return float64(time.Since(start).Nanoseconds() / time.Microsecond.Nanoseconds())
 }
+
+// ReportRateLimiterSaturation exposes saturation rate of rate limiter.
+// Metrics is collected every "interval".
+func ReportRateLimiterSaturation(rl util.RateLimiter, interval time.Duration) {
+	for {
+		BindingRateLimiterSaturation.Set(rl.Saturation())
```

Comment:
I guess we didn't really need to add another function but set metrics inside `util.Forever`.